test/externalTests/gp2.sh

```diff
@@ -31,12 +31,12 @@ BINARY_TYPE="$1"
 BINARY_PATH="$(realpath "$2")"
 SELECTED_PRESETS="$3"
 
-function compile_fn { npm run build; }
-function test_fn { npm test; }
+function compile_fn { yarn run build; }
+function test_fn { yarn test; }
 
 function gp2_test
 {
-    local repo="https://github.com/gnosis/gp-v2-contracts.git"
+    local repo="https://github.com/cowprotocol/contracts.git"
```

Comment:
Please update the URL in [`update_external_repos.sh`](https://github.com/ethereum/solidity/blob/develop/scripts/externalTests/update_external_repos.sh) as well.

Also, I think we should archive https://github.com/solidity-external-tests/gp-v2-contracts now and fork the new repo instead.

Reply:
Done. However, I do not have permissions to archive the old fork, but I created the new one there.

Reply:
OK, I'll archived them. I also gave you the `Owner` role in the org since going forward you'll probably be dealing with it quite a lot. You should now be able to do it yourself.

Also, when creating these repos please remember to disable github actions in them - we don't really want to run their tests and when they do, they often fail, sending annoying notifications. I also usually disable the wiki, since we don't use it and if we used one, it would be in the main repo anyway.